src/components/HathorHeader.js

```diff
@@ -78,10 +78,9 @@ const styles = StyleSheet.create({
   wrapper: {
     height: HEADER_HEIGHT,
     flexDirection: 'row',
-    alignItems: 'center',
-    borderColor: '#eee',
+    alignItems: 'flex-end',
```

Comment:
Reverts the central alignment to a bottom alignment because of the background color change.